core/src/main/java/com/exadel/aem/toolkit/core/lists/utils/ListPageUtils.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.exadel.aem.toolkit.core.lists.utils;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.sling.api.resource.ModifiableValueMap;
+import org.apache.sling.api.resource.ResourceResolver;
+import com.day.cq.commons.jcr.JcrUtil;
+import com.day.cq.wcm.api.Page;
+import com.day.cq.wcm.api.PageManager;
+import com.day.cq.wcm.api.WCMException;
+
+import com.exadel.aem.toolkit.core.CoreConstants;
+import com.exadel.aem.toolkit.core.lists.ListConstants;
+
+/**
+ * Contains methods for manipulation with List Page
+ */
+class ListPageUtils {
+
+    /**
+     * Creates a list page under given path
+     * @param resourceResolver Sling {@link ResourceResolver} instance used to create the list
+     * @param path             JCR path of the items list page.
+     * @return Created page containing list of entries or ${@code null} if {@link PageManager} cannot be instantiated
+     * or {@code path} is blank
+     * @throws WCMException if a page cannot be created
+     */
+    static Page createListPage(ResourceResolver resourceResolver, String path) throws WCMException {
```

Comment:
(very cosmetic) Just a lifehack -- when in a *non-public class*, a method is designed to be used outside the class, it is still better to make the method public. This will not change its visibility overall. But if someday you decide to re-structure code, and the whole class becomes public, then the method is up and ready. Else you'll probably need to add "public" twice